components/fieldgroup/stories/template.js

```diff
@@ -53,7 +70,20 @@ export const Template = (
 					[`${rootClass}InputLayout`]: true,
 				})}
 			>
-				${renderContent(items, { args: { isReadOnly, isRequired }, context })}
+				${inputType === "radio" ?
+					items.map((item) =>
+					Radio({
+					...item,
+					isReadOnly,
```

Comment:
We should probably add the `isRequired` arg back to the `Radio` and `Checkbox` components here.